src/sources/kubernetes_logs/mod.rs

```diff
@@ -87,33 +87,44 @@ const SELF_NODE_NAME_ENV_KEY: &str = "VECTOR_SELF_NODE_NAME";
 #[derive(Clone, Debug)]
 #[serde(deny_unknown_fields, default)]
 pub struct Config {
-    /// Specifies the label selector to filter `Pod`s with, to be used in addition to the built-in
-    /// `exclude` filter.
+    /// Specifies the [label selector][label_selector] to filter [Pods][pods] with, to be used in
+    /// addition to the built-in [exclude][exclude] filter.
+    ///
+    /// [label_selector]: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors
+    /// [pods]: https://kubernetes.io/docs/concepts/workloads/pods/
```

Comment:
Mega nit and in no way blocking but on the URLs you added, the trailing slash is not needed